### ISSUE TITLE
188. Log-Softmax & Cross-Entropy

### ISSUE THREAD

**Issue by preethamak** [Open-Deep-ML/DML-OpenProblem#560]
Added new quesion. log softmax and cross entropy for deep learning
